crafts.lua

```diff
@@ -17,4 +55,22 @@ if minetest.get_modpath("default") then
       {'default:mese_block', 'default:steelblock', 'default:mese_block'}
     }
   })
+
+  minetest.register_craft({
+    output = 'jumpdrive:warp_device',
+    recipe = {
+      {'technic:composite_plate', 'technic:wrought_iron_dust', 'technic:composite_plate'},
+      {'default:mese_block', 'technic:machine_casing', 'default:mese_block'},
+      {'technic:copper_coil', 'technic:hv_cable', 'technic:copper_coil'}
+    }
+  })
```

Comment:
Also if using warp device in crafting without technic it should use something that is available in mtg, for example:
```suggestion
  minetest.register_craft({
    output = 'jumpdrive:warp_device',
    recipe = {
      {'default:mese_crystal', 'default:diamond', 'default:mese_crystal'},
      {'default:mese_block', 'default:steelblock', 'default:mese_block'},
      {'default:mese_crystal', 'default:diamond', 'default:mese_crystal'}
    }
  })
```
Tried to make it less expensive but compared to technic counterpart this is prob too cheap already.